### ISSUE TITLE
Kind attribute omitted

### ISSUE THREAD

**Issue by shikolay** [cgiffard/Captionator#42]
Currently absence of kind attribute cause captionator to throw exception, but according to specs:
 if there is no kind attribute, implementation need consider this as 'subtitles' kind. 

From specs:
The attribute(kind) may be omitted. The missing value default is the subtitles state.

**Comment by cgiffard** [comment]
Yeah, this probably changed since I implemented the spec (ages and ages ago.)

I'll make this small change for you, but I am looking to entirely rewrite the project.
